contrib/config_regtest.toml

```diff
@@ -0,0 +1,11 @@
+daemon = false
```

Comment:
Can we name this file `config.toml` or `sample_config.toml` ? It has nothing to do with a Bitcoin network